File: week-02/day-01/draw_triangle.py
# Write a program that reads a number from the standard input, then draws a
# triangle like this:
#
# *
# **
# ***
# ****
#
# The triangle should have as many lines as the number was
lines = int(input("Number of lines: "))
stars = ""
for i in range(0, lines):
    for k in range(0, i+1):
        stars += "*"
    print (stars + "\n")
    stars = ""

File: week-07/day-01/draw-triangle.js
'use strict';

var lineCount = 4;

// Write a program that draws a
// triangle like this:
//
// *
// **
// ***
// ****
//
// The triangle should have as many lines as lineCount is

var stars = '';
for (let i = 0; i < lineCount; i++) {
  stars += "*";
  console.log(stars);
}